ProjectEureka/project-eureka-flutter
# PE-33-ImplmentLoginPage

**What I did:**
Implemented login and password widgets for login screen and signup screen. The user can now create an account and login with the same account or an existing account. Their info will be stored in Firebase.

**How to test:**

- Pull code from branch
- First thing you should see is the login screen.
 ![image](https://user-images.githubusercontent.com/38572314/106972021-e4bc2e00-6704-11eb-8712-194b579136a8.png)
You can either login with an existing account or create a new one.
- Click signup to create an account then enter an email and password.
- Both screens will redirect you the a basic homes screen and in your terminal the user uid will be displayed. ![image](https://user-images.githubusercontent.com/38572314/106972235-4ed4d300-6705-11eb-8085-427ec282d5ae.png)
- Basic Home Screen:  
![image](https://user-images.githubusercontent.com/38572314/106972248-54cab400-6705-11eb-9550-25ca61fb6734.png)
